app_dart/lib/src/request_handlers/postsubmit_luci_subscription.dart

```diff
@@ -80,9 +80,21 @@ class PostsubmitLuciSubscription extends SubscriptionHandler {
     }
     log.fine('Found $task');
 
+    // No need to process if
+    // 1) the build is `scheduled`. Task is marked as `In Progress`
```

Comment:
What is marking it as in-progress?

Reply:
Ah I see, the next time around this check will be made.

Reply:
1. For retry: https://github.com/flutter/cocoon/blob/main/app_dart/lib/src/service/luci_build_service.dart#L672
2. For scheduler: https://github.com/flutter/cocoon/blob/main/app_dart/lib/src/service/scheduler.dart#L147
3. For backfiller: https://github.com/flutter/cocoon/blob/main/app_dart/lib/src/request_handlers/scheduler/batch_backfiller.dart#L236